src/components/Articles/Articles.vue

```diff
@@ -5,22 +5,29 @@
       <label class="define-article-title">Articles</label>
     </div>
 
-    <div class="section-container" v-if="getBusinessInformation.hasRestrictions">
+    <div
+      class="section-container"
+      v-if="getBusinessInformation.hasRestrictions"
+      :class="{'invalid-section': invalidCompanyProvisions}">
       <company-provisions
         class="sub-section"
         :provisionsRemoved="getProvisionsRemoved"
         @companyProvisionsChanged="setProvisionsRemoved($event)"
         @haveChanges="emitHaveChanges($event)"
+        @onEditingCompanyProvisions="setEditingCompanyProvisions($event)"
```

Comment:
I was hoping the event would simply be called "isEditing" (and same for all components).

(Admittedly I would prefer the event on line 15 to be called "isChanged". But nvm that.)

The "on" naming is usually for event handlers, eg,
`@isEditing="onEditingDoSomething($event)"`

Reply:
ok. I can change to `isEditing`. 👍

Reply:
Thanks. That will be cleaner.

Also, other events can be "isValid", "isChanged" (or "hasChanges" or "hasChanged")... yeah we're still working on naming :)

Re: hardest problems in computer science - https://martinfowler.com/bliki/TwoHardThings.html